standard-covoiturage_openapi.yaml

```diff
@@ -116,7 +116,70 @@ paths:
                 type: integer
         500:
           description: Internal Server Error. Please try again later.
+    
+  /bookings:
+    post:
+      summary: |-
+        Sends new booking information of a passenger connected with a third-party provider back to the provider
+      description: |-
+        Route used to allow a carpool operator to send booking information to a third-party provider. This can be used in the context of a booking flow with deeplinking and when a passenger is using the Connect specification to link their accounts of a third-party service (e.g., a MaaS) and the operator. 
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CarpoolBooking'
+      responses:
+        201:
+          description: Successful operation.
+        400:
+          description: Bad Request. See error message.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    description: |-
+                      Explain why the request couldn't be processed.
+        401:
+          description: Unauthorized. You must authenticate.
+        409:
+          description: Conflict. This booking already exists.
+        429:
+          description: Too Many Requests. Please slow down.
+          headers:
+            Retry-After:
+              description: |-
+                How long to wait before making a new request (in seconds).
+              schema:
+                type: integer
+        500:
+          description: Internal Server Error. Please try again later.
 
+  /bookings/{bookingId}:
```

Comment:
I am more for a single `upsert` endpoint: 
- where the `id` of the booking is determined by the operator itself (shared id so easier cross-referential comparison, no need for the operator to store the MaaS platform internal ID for the booking, ...),
- that will prevent lifecycle issue (ie: concurrency between create and a very close update) and remove the need to handle status codes like 409 .
- that will not make assumptions on which attribute can change during update (and what must remain unchanged).